merge_command.go

```diff
@@ -180,3 +202,24 @@ func handleMergeConflict(issue Issue, issues Issues) *ErrorResponse {
 	}
 	return nil
 }
+
+func removeRemoteBranch(pr *github.PullRequest, gitRepos git.Repos) *ErrorResponse {
```

Comment:
Oh and https://github.com/salemove/github-review-helper/pull/28/commits/851378d155ac6f8bcbe53282945ad16c6fd1b355